### PR TITLE
Submit PAI-TF prediction job by alisa submitter

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: pkg/sql/alisa_submitter.go [hunk 1]
```diff
@@ -215,3 +189,34 @@ func writeFile(filePath, program string) error {
...
+	}
+	cfQuote := strconv.Quote(string(cfString))
+	ckpDir, err := pai.FormatCkptDir(modelName)
+	if err != nil {
+		return "", err
+	}
+
+	// submit table should format as: odps://<project>/tables/<table>,odps://<project>/tables/<table>...
+	parts := strings.Split(trainTable, ".")
+	submitTables := fmt.Sprintf("odps://%s/tables/%s", parts[0], parts[1])
```
Comment (weiguoz) [inline pkg/sql/alisa_submitter.go]: Is there an assertion `len(parts) >= 2` ?

File: pkg/sql/alisa_submitter.go [hunk 2]
```diff
@@ -215,3 +189,34 @@ func writeFile(filePath, program string) error {
 	f.WriteString(program)
 	return nil
 }
+
+func getPAIcmd(cc *pai.ClusterConfig, modelName, trainTable, valTable, resTable string) (string, error) {
+
```
Comment (typhoonzero) [inline pkg/sql/alisa_submitter.go:200]: can delete this blank line?

**Review by weiguoz — DISMISSED** [review]
LGTM

**Review by typhoonzero — DISMISSED** [review]
A tiny comment

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**